src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -352,6 +356,37 @@ private void EmitExpressionCore(BoundExpression expression, bool used)
             }
         }
 
+        private void EmitLoweredIsPatternExpression(BoundLoweredIsPatternExpression node, bool used)
+        {
+            DefineLocals(node.Syntax, node.Locals);
+#if !DEBUG
+            EmitStatements(node.Statements);
+#else
+            foreach (BoundStatement statement in node.Statements)
+            {
+                EmitStatement(statement, _builder.GetStackDepth());
```

Comment:
>_builder.GetStackDepth() [](http://example.com/codeflow?start=41&length=24)

Do we expect this to change in the process? Otherwise, consider capturing it in a local outside of the loop.